examples/proc_combined_ODNP.py

```diff
@@ -0,0 +1,332 @@
+import os, time, h5py
+import pylab as plt
+from numpy import empty
+from sympy import exp as s_exp
+from sympy import symbols, Symbol
+from matplotlib.ticker import FuncFormatter
+import matplotlib.transforms as transforms
+from pyspecdata import *
+from pyspecProcScripts import *
+from pyspecProcScripts import lookup_table
+from Instruments.logobj import logobj
+logger = init_logging("info")
+Ep_signal_pathway = {'ph1':1}
+fl=fl_mod()
+filename='211213_10mM_TEMPOL_test'
+file_location = 'ODNP_NMR_comp/ODNP'
+Ep_f_slice=(0.15e3,0.4e3)
+IR_f_slice=(0.15e3,0.4e3)
+Ep_t_range = (0,0.2)
+t_range = (0,0.2)
+excluded_pathways = [(0,0)]
+R1w = 1/2.172
+nPowers = 15
+ppt = 1.5167e-3
+C=0.01
+#{{{load in log
+with h5py.File(search_filename(filename+".h5",exp_type='ODNP_NMR_comp/ODNP',unique=True),'r') as f:
+    log_grp = f['log']
+    thislog = logobj()
+    thislog.__setstate__(log_grp)
+    read_array = thislog.total_log
+    read_dict = thislog.log_dict
+for j in range(len(read_array)):
+    thistime,thisrx,thispower,thiscmd = read_array[j]
```

Comment:
you're looping over an array, which is never good -- what is `read_array` -- is it a structured array?

Reply:
you can answer this question by looking at `read_array.dtype`.  If it is a structured array, what are the fields?  I can give you the code to replace this.

Reply:
I took this from the test_power_control_server_read.py example in inst_notebooks. After printing read_array it is an ndarray and looks like a structured array 
`[(1.63960294e+09, -1. , -10.095 ,  8060245099936175237)
 (1.63960294e+09, 41.8,   7.3434, -8640926761156270880)
 (1.63960294e+09, 42.4,   7.3496, -8640926761156270880) ...
 (1.63960717e+09, 98.2,   6.2436,                    0)
 (1.63960717e+09, 98.6,   6.2437,                    0)
 (1.63960717e+09, 98.6,   6.2436, -7387592744514551751)]`

Reply:
It didn't print the field names though so IDK as though there are any?

Reply:
yes -- as I say above, you should print `read_array.dtype` to get the field names, then, to answer this question.

Reply:
`[('time', '<f8'), ('Rx', '<f8'), ('power', '<f8'), ('cmd', '<i8')]` is the dtype for read_array

Reply:
OK, it's a structured array, then.  So, if you wanted an array of the times, you would just do `timearray = readarray['time']`, for example.  But you don't use this, anways, so I'm deleting it.